next.config.js

```diff
@@ -1,5 +1,9 @@
 /** @type {import('next').NextConfig} */
+
```

Comment:
1) Can you update this file to TS?
2) The comment at the top of file, is it necessary?

Reply:
next.config.ts isn't supported by NextJs themselves yet. You can learn more here https://stackoverflow.com/questions/65319081/error-configuring-next-js-via-next-config-ts-is-not-supported-please-replace

Reply:
Ok